src/js/components/SearchResults/ResultsTable.tsx

```diff
@@ -82,14 +81,10 @@ export default function ResultsTable(props: Props) {
     overScan: 1
   })
 
-  useDebouncedEffect(
-    () => {
-      if (selection.isEmpty()) return
-      dispatch(viewLogDetail(logs[selection.currentRange[0]]))
-    },
-    400,
-    [selection]
-  )
+  useEffect(() => {
+    if (selection.isEmpty()) return
+    dispatch(viewLogDetail(logs[selection.currentRange[0]]))
+  }, [selection])
```

Comment:
Removed the delay here. Things feel much snappier now and it doesn't look like there was a performance hit other places.